non-wheel/httpcrtsrv/httpsrv.py

```diff
@@ -0,0 +1,444 @@
+</Directory>
+#End XML Toolkit http settings
+
+# keep access logs 10 days, error logs 10 days, FastCGI logs 10 days
+LogMaint /www/%s/logs/access_log 10 0
+LogMaint /www/%s/logs/error_log 10 0
+LogMaint /www/%s/logs/error_zfcgi 10 0
+
+# Maintain Logs at 3 am (0 = midnight, 23 = 11 pm, etc)  
+# Set for an hour when the server is active (i.e. not during an IPL or backup)  
+LogMaintHour 3
+
+IncludeOptional /www/%s/conf/apache-sites/*.conf
+''' % (port, name, name, name, name, name, name)
+
+
+def get_zendphp7_conf(name, port):
+    return '''LoadModule proxy_module /QSYS.LIB/QHTTPSVR.LIB/QZSRCORE.SRVPGM
+LoadModule proxy_http_module /QSYS.LIB/QHTTPSVR.LIB/QZSRCORE.SRVPGM
+LoadModule proxy_connect_module /QSYS.LIB/QHTTPSVR.LIB/QZSRCORE.SRVPGM
+LoadModule proxy_ftp_module /QSYS.LIB/QHTTPSVR.LIB/QZSRCORE.SRVPGM
+LoadModule zend_enabler_module /QSYS.LIB/QHTTPSVR.LIB/QZFAST.SRVPGM
+DefaultFsCCSID 37 
+CGIJobCCSID 37    
+
+Listen *:%s
+DocumentRoot /www/%s/htdocs
+DirectoryIndex index.php index.html
+
+Options -ExecCGI -FollowSymLinks -SymLinksIfOwnerMatch -Includes -IncludesNoExec -Indexes -MultiViews
+LogFormat "%%h %%l %%u %%t \\"%%r\\" %%>s %%b \\"%%{Referer}i\\" \\"%%{User-Agent}i\\"" combined
+LogFormat "%%{Cookie}n \\"%%r\\" %%t" cookie
+LogFormat "%%{User-agent}i" agent
+LogFormat "%%{Referer}i -> %%U" referer
+LogFormat "%%h %%l %%u %%t \\"%%r\\" %%>s %%b" common
+CustomLog logs/access_log combined
+SetEnvIf "User-Agent" "Mozilla/2" nokeepalive
+SetEnvIf "User-Agent" "JDK/1\.0" force-response-1.0
+SetEnvIf "User-Agent" "Java/1\.0" force-response-1.0
+SetEnvIf "User-Agent" "RealPlayer 4\.0" force-response-1.0
+SetEnvIf "User-Agent" "MSIE 4\.0b2;" nokeepalive
+SetEnvIf "User-Agent" "MSIE 4\.0b2;" force-response-1.0
+
+TimeOut 30000
+KeepAlive On
+KeepAliveTimeout 5
+HotBackup Off
+
+#AddCharset UTF-8       .utf8
+#AddCharset utf-8       .utf8
+#AddCharset utf-7       .utf7
+AddCharset UTF-8 .htm .html .xml
+
+# zend fastcgi
+AddType application/x-httpd-php .php
+AddHandler fastcgi-script .php
+
+RewriteEngine on 
+
+<Directory />
+   Order Deny,Allow 
+   Deny From all     
+</Directory>
+
+<Directory /www/%s/htdocs>
+  Options FollowSymLinks 
+  order allow,deny
+  allow from all
+  AllowOverride all
+</Directory>
+
+#XML Toolkit http settings
+ScriptAlias /cgi-bin/ /QSYS.LIB/zendphp7.LIB/
+<Directory /QSYS.LIB/zendphp7.LIB/>
+    AllowOverride None
+    order allow,deny
+    allow from all
+    SetHandler cgi-script
+    Options +ExecCGI
+</Directory>
+#End XML Toolkit http settings
+
+# keep access logs 10 days, error logs 10 days, FastCGI logs 10 days
+LogMaint /www/%s/logs/access_log 10 0
+LogMaint /www/%s/logs/error_log 10 0
+LogMaint /www/%s/logs/error_zfcgi 10 0
+
+# Maintain Logs at 3 am (0 = midnight, 23 = 11 pm, etc)  
+# Set for an hour when the server is active (i.e. not during an IPL or backup)  
+LogMaintHour 3
+
+IncludeOptional /www/%s/conf/apache-sites/*.conf
+''' % (port, name, name, name, name, name, name)

```

Comment:
Might want a healthy dose of [pyformat](https://pyformat.info/) throughout the file, but the `%%` get really bad here. Really, though, Python 3.6 and [f-strings](https://www.python.org/dev/peps/pep-0498/) can't come soon enough!